src/components/CourseTable/CourseTableCard/CourseTableCard.tsx

```diff
@@ -125,6 +145,16 @@ function CourseTableCard({
   const saveChanges = async () => {
```

Comment:
I can't remember if there is any reason why I wrote this because I wrote this almost one year ago. But it seems like I wrote stupid & unnecessary code at that time. Thanks for correcting me.